### PR TITLE
Issuenumber5/#135/result list of ranking lists

### PR DESCRIPTION
**Pull request by pawelsoundso**
Adds a list of participants of ranking lists, with an initial value on join and a button, which should be modified later by #272.

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -82,10 +82,27 @@
...
+    </tr>
+    </thead>
     <tbody>
-    <% @event.teams.each do |participant| %>
-      <tr><td><%= link_to_participant participant %></td></tr>
+    <% @event.participants.order(rating: :desc).each do |participant| %>
+      <tr><td><%= link_to_participant participant.team %></td>
+        <% if @event.is_a? Rankinglist %>
+          <td><%= participant.rating %></td>
+          <% if can? :leave, @event %>
```
Comment (sumpfranze) [inline app/views/events/show.html.erb]: is leave the correct action to test whether a match can be added or not?

**Review by sumpfranze — COMMENTED** [review]
otherwise lgtm

File: app/views/events/show.html.erb (hunk 1 above)
Comment (Jeilef) [reply app/views/events/show.html.erb]: this is implemented in #272

**Review by Jeilef — APPROVED** [review]
like Jacob, Looks Good To Merge

**Pull request merged**